cli/setup-project.js

```diff
@@ -123,6 +123,19 @@ const updateGitHubWorkflows = (projectName) => {
       newFileName: '.github/workflows/new-app-version.yml',
     },
   ]);
+
+  // Update Pull Request Template
+  projectFilesManager.replaceFilesContent([
+    {
+      fileName: '.github/PULL_REQUEST_TEMPLATE.md',
+      replacements: [
+        {
+          searchValue: /^[\s\S]*?(?=## What does this do\?)/,
+          replaceValue: '',
```

Comment:
shouldn't we replace it with What does this do?

Reply:
The regex matches the beginning of the text until it reaches the text "## What does this do", but it doesn't include it in the match.

<img width="825" alt="image" src="https://github.com/user-attachments/assets/3568da6c-0927-4090-a048-3851eb02caf6">